### PR TITLE
DDF-3066 Align Checkboxes in System Section of Admin Console

### PR DESCRIPTION
**Pull request by josephthweatt**
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged)?
@mcalcote 
@djblue 
#### Select at least one member from relevant component team(s) from below.
[UI](https://github.com/orgs/codice/teams/ui)
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@clockard 
@lessarderic 
#### How should this be tested? (List steps with links to updated documentation)
1. Build ddf.
2. Open Admin Console, select System.
3. Click on various configuration names. In the modal, the checkboxes should be aligned as shown in the picture below.
4. Repeat steps 2-3 for Firefox, Chrome, and IE10
#### What are the relevant tickets?
[DDF-3066](https://codice.atlassian.net/browse/)
#### Screenshots
screen shot 2017-06-09 at 12 31 24 pm

#### Checklist:
- [ ] Documentation Updated
- [ ] Change Log Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

File: platform/admin/ui/src/main/webapp/templates/configuration/configurationItem.handlebars [hunk 1]
```diff
@@ -14,7 +14,7 @@
  
 <div class="{{id}} metatype control-group">
     {{#is type 11}}
-            <label class="checkbox">
+            <label class="checkbox" style="margin-left:20px;">
```
Comment (clockard) [inline platform/admin/ui/src/main/webapp/templates/configuration/configurationItem.handlebars]: the style should probably be set in the css file.

Comment (josephthweatt) [inline platform/admin/ui/src/main/webapp/templates/configuration/configurationItem.handlebars]: @clockard Okay, do you know which css file would correspond to this file?

**Comment by josephthweatt** [comment]
build now

**Comment by josephthweatt** [comment]
build now

**Pull request merged**